### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by izhuxiaoqing**

### CODE REVIEW

**Review by whitewum — APPROVED** [review]

File: docs-2.0/README.md [hunk 1]
```diff
@@ -6,14 +6,43 @@ Nebula Graph is a distributed, scalable, and lightning-fast graph database.
...
+
+## Graphic Illustrations
+
+!!! note
+
+    Additional information or operation-related notes.
+
+!!! caution
+
+    Cautions that need strict observation. If not, systematic breakdown, data loss and security issue may happen.
```
Comment (randomJoe211) [inline docs-2.0/README.md:32]: data loss, and

Add a comma before and.

Comment (randomJoe211) [reply docs-2.0/README.md:32]: security issues?

File: docs-2.0/README.md [hunk 2]
```diff
@@ -6,14 +6,43 @@ Nebula Graph is a distributed, scalable, and lightning-fast graph database.
...
+
+!!! danger
+
+    Operations that may cause danger. If not observed, systematic breakdown, data loss and security issue will happen.
```
Comment (randomJoe211) [inline docs-2.0/README.md:36]: * A comma before and.
* issues

**Review by randomJoe211 — APPROVED** [review]
👍

**Review by cooper-lzy — APPROVED** [review]

**Pull request merged**